resources/services/backup/global_settings.go

```diff
@@ -0,0 +1,59 @@
+package backup
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource global_settings --config gen.hcl --output .
+func GlobalSettings() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_backup_global_settings",
+		Resolver:     fetchBackupGlobalSettings,
+		Multiplex:    client.AccountMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "global_settings",
+				Description: "The status of the flag isCrossAccountBackupEnabled.",
+				Type:        schema.TypeJSON,
+			},
```

Comment:
Why is this a JSON field? By the description, it seems like it should be a boolean.